### PR TITLE
fix typo in ontime.md

### PR DESCRIPTION
**Pull request by bi122**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category:
- Documentation

### CODE REVIEW

File: docs/en/getting_started/example_datasets/ontime.md [hunk 1]
```diff
@@ -320,13 +320,13 @@ ORDER BY Year;
...
-```sql
-SELECT DestCityName, uniqExact(OriginCityName) AS u F
-ROM ontime
-WHERE Year>=2000 and Year<=2010
-GROUP BY DestCityName
-ORDER BY u DESC
-LIMIT 10;
+``` sql
+SELECT DestCityName, uniqExact(OriginCityName) AS u 
+FROM ontime
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/ontime.md]:
```suggestion
FROM ontime
```

File: docs/en/getting_started/example_datasets/ontime.md [hunk 2]
```diff
@@ -320,13 +320,13 @@ ORDER BY Year;
...
-ROM ontime
-WHERE Year>=2000 and Year<=2010
-GROUP BY DestCityName
-ORDER BY u DESC
-LIMIT 10;
+``` sql
+SELECT DestCityName, uniqExact(OriginCityName) AS u 
+FROM ontime 
+WHERE Year >= 2000 and Year <= 2010 
+GROUP BY DestCityName
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/ontime.md]:
```suggestion
GROUP BY DestCityName
```

File: docs/en/getting_started/example_datasets/ontime.md [hunk 3]
```diff
@@ -320,13 +320,13 @@ ORDER BY Year;
...
-SELECT DestCityName, uniqExact(OriginCityName) AS u F
-ROM ontime
-WHERE Year>=2000 and Year<=2010
-GROUP BY DestCityName
-ORDER BY u DESC
-LIMIT 10;
+``` sql
+SELECT DestCityName, uniqExact(OriginCityName) AS u 
+FROM ontime 
+WHERE Year >= 2000 and Year <= 2010
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/ontime.md]:
```suggestion
WHERE Year >= 2000 and Year <= 2010
```

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by blinkov** [comment]
@bi122 those extra spaces were not removed

**Review by blinkov — APPROVED** [review]

File: docs/en/getting_started/example_datasets/ontime.md [hunk 4]
```diff
@@ -320,13 +320,13 @@ ORDER BY Year;
 
 Q8. The most popular destinations by the number of directly connected cities for various year ranges
 
-```sql
-SELECT DestCityName, uniqExact(OriginCityName) AS u F
-ROM ontime
-WHERE Year>=2000 and Year<=2010
+``` sql
```
Comment (alexey-milovidov) [inline docs/en/getting_started/example_datasets/ontime.md]: This whitespace - it is ok?

**Pull request merged**